.github/issue-templates/packages-out-of-sync.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: `light-the-torch` is out of sync with the PyTorch package indices
```

Comment:
Not sure if this is actually the problem, but it seems a leading backtick is not supported:

https://github.com/pmeier/light-the-torch/actions/runs/4793682109/jobs/8526354021

See JasonEtco/create-an-issue#155